jenkins/debian/debos/stretch.yaml

```diff
@@ -68,6 +70,15 @@ actions:
     chroot: true
     command: rm /usr/lib/tmpfiles.d/dbus.conf
 
+  - action: run
+    description: update-initramfs step
+    chroot: true
+    script: scripts/create_tiny_ramdisk.sh
+
+  - action: run
+    chroot: false
+    command: mv ${ROOTDIR}/boot/initrd.img-min ${ARTIFACTDIR}/{{ $basename -}}/tiny.rootfs.cpio.gz
```

Comment:
minor nit: I propose we s/tiny.rootfs/initrd/.  IOW, mv initrd.img-min -> initrd.cpio.gz